ticket/src/main/resources/application.yml

```
@@ -1,7 +1,6 @@
 spring:
   application:
-    name: ticket
+    name: ticket-service
 
-logging:
-  level:
-    root: WARN
\ No newline at end of file
+  config:
+    import: configserver:${CONFIG_SERVER_URL:http://localhost:19000}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)